lib/nexaas/throttle/guardian.rb

```diff
@@ -42,6 +43,10 @@ def extensions_regexp
           /\.(#{extensions.join("|")})/
         end
       end
+
+      def ignore_user_agents?
+        ignored_user_agents && ignored_user_agents.include?(request.user_agent)
```

Comment:
this validation should be less strict. currently it requires a very specific user-agent to be expected, when in reality user-agent strings vary a lot, mostly by version numbers.

ideally, instead of expecting a string to match completely, we could receive a regex to match to the received user-agent.